NuGet.config

```diff
@@ -5,5 +5,6 @@
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />
     <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="Roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
We should add this to our mirror build instead of adding a new feed. cc @kichalla